compute_endpoint/tests/unit/test_htex_regression.py

```diff
@@ -0,0 +1,95 @@
+import concurrent.futures
+import logging
+import multiprocessing
+import random
+import uuid
+
+import pytest
+from globus_compute_common import messagepack
+from globus_compute_endpoint.engines import HighThroughputEngine
+from globus_compute_sdk.serialize import ComputeSerializer
+from tests.utils import double, ez_pack_function
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture
+def engine():
+    ep_id = uuid.uuid4()
+    engine = HighThroughputEngine(
+        label="HTEXEngine",
+        heartbeat_period=1,
+        worker_debug=True,
+    )
+    queue = multiprocessing.Queue()
+    engine.start(endpoint_id=ep_id, run_dir="/tmp", results_passthrough=queue)
+
+    yield engine
+    engine.shutdown()
```

Comment:
I've updated this to use the regular old `queue` module and I've moved this to `integration` tests because this is indeed too "chunky" for unit tests.